docs/src/extend/custom-rules.md

```diff
@@ -214,37 +228,37 @@ context.report({
 });
 ```
 
-The node contains all of the information necessary to figure out the line and column number of the offending text as well the source text representing the node.
+The node contains all the information necessary to figure out the line and column number of the offending text as well as the source text representing the node.
 
-### Using message placeholders
+#### Using Message Placeholders
 
 You can also use placeholders in the message and provide `data`:
 
 ```js
-{% raw %}
 context.report({
     node: node,
     message: "Unexpected identifier: {{ identifier }}",
     data: {
         identifier: node.name
     }
 });
-{% endraw %}
```

Comment:
If we remove `{% raw %}` ... `{% endraw %}`, the templating engine will interpret `{{ identifier }}` as the syntax to display a variable, so we'll not get the output we want.

Before this change:

![image](https://user-images.githubusercontent.com/44349756/223121942-58d4daa5-8823-46d8-a706-eeba8e4bfe4c.png)


After this change (deploy preview):

![image](https://user-images.githubusercontent.com/44349756/223121320-c2f7d9fb-a93f-4885-bf3f-a4ba57faeafd.png)

The same applies to other places where  `{% raw %}` ... `{% endraw %}` is removed.

Reply:
will add back, thx for clarifying